### PR TITLE
VDPN-11 Admin: notify logged-in users about offers for objectives on their wishlist

### PR DESCRIPTION
**Pull request by ancachetan**
…otification using an alert

### CODE REVIEW

File: TravelPoints-UI/src/index.html [hunk 1]
```diff
@@ -10,6 +10,11 @@
         integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <script>
+    if (global === undefined) {
+      var global = window;
+    }
+  </script>
```
Comment (iDietzer) [inline TravelPoints-UI/src/index.html:17]: why is this needed?

File: TravelPoints-UI/src/app/services/websocket.service.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import * as Stomp from 'stompjs';
+import SockJS from 'sockjs-client';
+import {Observable} from 'rxjs';
+import {Notification} from "../models/Notification";
+
+@Injectable({
+  providedIn: 'root',
+})
+export class WebsocketService {
+  baseUrl: string = 'http://localhost:8888/websocket'
```
Comment (iDietzer) [inline TravelPoints-UI/src/app/services/websocket.service.ts]: add readonly to mark the baseUrl as a constant

File: TravelPoints-UI/src/app/components/navigation-bar/navigation-bar.component.ts [hunk 3]
```diff
@@ -46,9 +52,10 @@ export class NavigationBarComponent implements OnInit{
         alert("LogOut successful!")
         localStorage.clear()
         this.token = null
```
Comment (iDietzer) [inline TravelPoints-UI/src/app/components/navigation-bar/navigation-bar.component.ts]: make the token of type string | undefined instead of string | null to respect the typescript coding guidelines

**Review by iDietzer — CHANGES_REQUESTED** [review]

File: TravelPoints-UI/src/index.html (hunk 1 above)
Comment (ancachetan) [reply TravelPoints-UI/src/index.html:17]: I received an error if not adding it:
Uncaught ReferenceError: global is not defined

**Review by iDietzer — APPROVED** [review]
